### PR TITLE
Add laplace noise

### PR DESCRIPTION
**Pull request by EnLAI111**

### CODE REVIEW

File: solvers/ADMM.py [hunk 1]
```diff
@@ -0,0 +1,80 @@
+from benchopt import BaseSolver
+from benchopt.stopping_criterion import SufficientDescentCriterion
+from benchopt import safe_import_context
```
Comment (agramfort) [inline solvers/ADMM.py]: you add this new solver on the laplace PR on purpose?

Comment (tomMoral) [reply solvers/ADMM.py]: It is an issue of rebasing

**Comment by EnLAI111** [comment]
It doesn't check the tests automatically.

**Comment by agramfort** [comment]
@EnLAI111 maybe try to push a blank commit

**Comment by EnLAI111** [comment]
> @EnLAI111 maybe try to push a blank commit

The tests are still not be checked.

**Comment by agramfort** [comment]
it's working now

**Pull request merged**